extensions/ql-vscode/src/extension.ts

```diff
@@ -1875,6 +1852,36 @@ async function compileAndRunQueryOnMultipleDatabases(
   }
 }
 
+async function previewQueryHelp(
+  cliServer: CodeQLCliServer,
+  selectedQuery: Uri,
+  qhelpTmpDir: DirResult,
```

Comment:
I think this would be better if the `selectedQuery` was the last argument since it's the argument to the command. We now have (from extension, from user, from extension), while I would expect everything from the extension to be grouped together.

Reply:
Good point. I've swapped these two around.